### PR TITLE
TE-2192 handleErrors returns any status code when response is not ok

### PR DESCRIPTION
**Pull request by red-runner**
[Related YouTrack issue](https://youtrack.lodgify.net/issue/TE-2192)

### What **one** thing does this PR do?
Allows handle errors to return any `statusCode` when the response is not ok.

### Log text/plain
text:plain

### Log application/json
json

### CODE REVIEW

File: src/utils/handleErrors.js [hunk 1]
```diff
@@ -6,11 +6,15 @@
...
   if (!response.ok) {
-    if (response.status >= 500) {
-      // eslint-disable-next-line no-console
-      console.error(response);
-    }
-    throw Error(response.statusText);
+    // eslint-disable-next-line no-console
+    console.log(response.status);
+    // eslint-disable-next-line no-console
+    console.log(response.statusText);
```
Comment (Jonnygwi) [inline src/utils/handleErrors.js]: I think these should be `console.errors`. What do you think?

File: src/utils/handleErrors.js [hunk 2]
```diff
@@ -6,11 +6,24 @@
  */
 export const handleErrors = response => {
   if (!response.ok) {
-    if (response.status >= 500) {
-      // eslint-disable-next-line no-console
-      console.error(response);
-    }
-    throw Error(response.statusText);
+    const { status, statusText, url } = response;
+    const body = JSON.stringify(response.body, null, ' ');
```
Comment (marcdavi-es) [inline src/utils/handleErrors.js]: From the animation in the description, it looks like we're logging the `ReadableStream` which wraps the body content. This isn't informative. We need to access the content of the body using one of the methods documented here (https://developer.mozilla.org/en-US/docs/Web/API/Body)

File: src/utils/handleErrors.js (hunk 1 above)
Comment (red-runner) [reply src/utils/handleErrors.js]: In the end I spoke to @marcdavi-es IRL and we decided to leave it as `console.log`

**Pull request merged**